src/main/java/org/opensearch/security/configuration/SecurityIndexSearcherWrapper.java

```diff
@@ -39,6 +39,7 @@
 import org.opensearch.core.common.transport.TransportAddress;
```

Comment:
Can we rename this class to SystemIndexSearcherWrapper?

Reply:
Done in: [7402adb](https://github.com/opensearch-project/security/pull/4774/commits/7402adb5f16740e05cabfc211534e13a24985ac9)